### PR TITLE
Analytics

### PR DESCRIPTION
**Pull request by blackpanther1881**

### CODE REVIEW

File: .env.local [hunk 1]
```diff
@@ -6,4 +6,5 @@ REACT_APP_CHAIN_ID=test-core-1
 REACT_APP_CHAIN_NAME="Persistence test-net"
 REACT_APP_WEBSITE_URL=https://test-core-1.wallet.persistence.one
 REACT_APP_IBC_CONFIG="ibcStaging.json"
-REACT_APP_SENTRY_API=""
\ No newline at end of file
+REACT_APP_SENTRY_API=""
+REACT_APP_GA_TRACKING_ID= "UA-2191745621-1"
```
Comment (puneet2019) [inline .env.local:10]: empty?

Comment (blackpanther1881) [reply .env.local:10]: this is some random key. empty also work

File: src/config.json [hunk 2]
```diff
@@ -20,6 +20,8 @@
   "ibcRemoteHeightIncrement": 150,
   "ibcDefaultPort": "transfer",
   "maxAccountIndex": 2147483647,
+  "stagingTrackingID": "UA-219125621-1",
```
Comment (puneet2019) [inline src/config.json:23]: using these too?

Comment (blackpanther1881) [reply src/config.json]: removed

**Review by puneet2019 — APPROVED** [review]

**Pull request merged**